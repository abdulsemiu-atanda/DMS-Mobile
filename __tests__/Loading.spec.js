import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import Loading from '../components/shared/Loading.react'

describe('Loading', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Loading animating />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
