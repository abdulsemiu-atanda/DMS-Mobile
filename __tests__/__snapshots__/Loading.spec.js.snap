// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="#01f0b3"
  hidesWhenStopped={true}
  size="large"
/>
`;
